hnn_core/viz.py

```diff
@@ -4,6 +4,7 @@
 #          Sam Neymotin <samnemo@gmail.com>
 
 import numpy as np
+from itertools import cycle
```

Comment:
I wasn't quite sure if `cycler` is installed with `maplotlib`? Went with itertools as a safer option because it's a Python builtin.